# HG changeset patch
# User Rik <rik@octave.org>
# Date 1471187585 25200
#      Sun Aug 14 08:13:05 2016 -0700
# Node ID c83deaa3fbe612d3c7fc4fbe98db73b1a7f73e3e
# Parent  ed023556d4facec983e70cd11560e668fbad6b86
build: Fix wrong private/ directory for __rotate_around_axis__.m (cset 57fded74ee22).

* scripts/plot/draw/module.mk: Add private/ to directory for
__rotate_around_axis__.m.

diff --git a/scripts/plot/draw/module.mk b/scripts/plot/draw/module.mk
--- a/scripts/plot/draw/module.mk
+++ b/scripts/plot/draw/module.mk
@@ -11,17 +11,17 @@ scripts_plot_draw_PRIVATE_FCN_FILES = \
   scripts/plot/draw/private/__ezplot__.m \
   scripts/plot/draw/private/__interp_cube__.m \
   scripts/plot/draw/private/__line__.m \
   scripts/plot/draw/private/__marching_cube__.m \
   scripts/plot/draw/private/__patch__.m \
   scripts/plot/draw/private/__pie__.m \
   scripts/plot/draw/private/__plt__.m \
   scripts/plot/draw/private/__quiver__.m \
-  scripts/plot/draw/__rotate_around_axis__.m \
+  scripts/plot/draw/private/__rotate_around_axis__.m \
   scripts/plot/draw/private/__scatter__.m \
   scripts/plot/draw/private/__stem__.m \
   scripts/plot/draw/private/__unite_shared_vertices__.m
 
 scripts_plot_draw_FCN_FILES = \
   scripts/plot/draw/area.m \
   scripts/plot/draw/barh.m \
   scripts/plot/draw/bar.m \
